apollo-router/tests/integration_tests.rs

```diff
@@ -252,7 +252,15 @@ async fn query_rust(
     let stream = router.ready().await.unwrap().call(request).await.unwrap();
     let (_, response) = stream.response.into_parts();
 
-    (response, counting_registry)
+    match response {
+        ResponseBody::GraphQL(response) => (response, counting_registry),
+        ResponseBody::RawJSON(_) => {
+            panic!("Expected graphql response")
+        }
+        ResponseBody::RawString(_) => {
+            panic!("Expected graphql response")
+        }
```

Comment:
NIT: It might be neater to have:
```
    match response {
        ResponseBody::GraphQL(response) => (response, counting_registry),
        _ => panic!("Expected graphql response"),
    }
```
?

Reply:
Fixed